pygmt/src/fitcircle.py

```diff
@@ -0,0 +1,127 @@
+"""
+fitcircle - Find mean position and great [or small] circle fit to points on sphere.
+"""
+import warnings
+
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile, build_arg_string, fmt_docstring, use_alias
+
+
+@fmt_docstring
+@use_alias(
+    L="norm",
+    S="small_circle",
+    V="verbose",
+)
+def fitcircle(data, output_type="pandas", outfile=None, **kwargs):
+    r"""
+    Find mean position and great [or small] circle fit to points on sphere.
+
+    **fitcircle** reads lon,lat [or lat,lon] values from the first two
+    columns on standard input [or *table*]. These are converted to
```

Comment:
Standard input (stdin) won't be needed here.
```suggestion
    columns of the table. These are converted to
```